core/domain/feedback_services.py

```diff
@@ -135,6 +173,16 @@ def create_message(
 
 
 def _get_message_from_model(message_model):
+    """Convert the FeedbackMessageModel to a FeedbackMessage
+
+    Args:
+        message_model: FeedbackMessageModel. The FeedbackMessageModel to
+            be converted
+
+    Returns:
+        FeedbackMessage. The FeedbackMessage converted from the
```

Comment:
Maybe: "The resulting FeedbackMessage domain object." -- as opposed to repeating the previous thing.